.editorconfig

```diff
@@ -1,10 +0,0 @@
-root = true
```

Comment:
Re-add this and check if biome reads it.

Reply:
I re-introduced `.editorconfig` and instructed biome to use that instead
[`a75287d` (#10)](https://github.com/trivikr/aws-sdk-js-otel-test/pull/10/commits/a75287df9b730e3f65ac9d2d4ddb4abf04c0ed3a)